jupyterhub_moss/spawner.py

```diff
@@ -79,6 +78,12 @@ def _validate_partitions(self, proposal):
                 env.setdefault("add_to_path", True)
         return partitions
 
+    slurm_info_cmd = traitlets.Unicode(
+        # Get number of nodes and cores for all partitions
+        r"sinfo -a -N --noheader -o \'%R %C %m\'",
```

Comment:
```suggestion
        r"sinfo -a -N --noheader -o '%R %t %m'",
```

This is too keep the same command as before and `\'` doesn't seems to work.

Reply:
you're right, that was an unintended change. We are showing idle cores instead of idle nodes in the table of available resources. Since notebooks do not use full nodes, we think it is more useful for the user to know about cores. If you are interested in this change I will open another PR for it.

Reply:
Indeed, the max number of idle cores available on a node would be more appropriate, PR welcomed!